Translate React Conf Recap. (#165)
reactjs/pt-BR.reactjs.org: content/blog/2018-11-13-react-conf-recap.md

---

> title: "Recapitulando o React Conf: Hooks, Suspense, e Renderização Simultânea"

Reviewer: ```suggestion
title: "Recapitulando a React Conf: Hooks, Suspense, e Renderização Concorrente"
```

Eu acho que Conf seria no feminino, para manter a coerência com "conferência".

E o termo "concorrente" para simultânea já é consagrado em computação, então acho que pode ser usado, sem problemas.

Author: Pois é eu também fiquei um pouco na dúvida. Mas obrigado!

author: [tomocchino]
---

> Neste ano o [React Conf](https://conf.reactjs.org/) aconteceu nos dias 25 e 26 de Outubro em Henderson, Nevada, onde mais de 600 participantes se reuniram para discutir sobre as últimas novidades em engenharia de UI.

Reviewer: ```suggestion
Neste ano a [React Conf](https://conf.reactjs.org/) aconteceu nos dias 25 e 26 de Outubro em Henderson, Nevada, onde mais de 600 participantes se reuniram para discutir sobre as últimas novidades em engenharia de UI.
```

<br>

<iframe width="560" height="315" src="https://www.youtube.com/embed/V-QO-KO90iQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Sophie Alpert e Dan Abramov começaram o primeiro dia com sua palestra: React Today and Tomorrow (React de hoje e o de amanhã). Na palestra, eles deram uma introdução sobre [Hooks](/docs/hooks-intro.html), que são uma nova proposta de adicionar a capacidade de acessar recursos como o state (estado) sem a necessidade de escrever uma classe JavaScript. Hooks prometem simplificar drasticamente o código necessário para os componentes do React e atualmente estão disponíveis em uma versão alpha do React.

<br>

<iframe width="560" height="315" src="https://www.youtube.com/embed/ByBPyMBTzM0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

> Na manhã do segundo dia, Andrew Clark e Brian Vaughn apresentaram Concurrent Rendering in React (Renderização Simultânea em React). Andrew abordou a API recentemente anunciada [React.lazy](/blog/2018/10/23/react-v-16-6.html) para code splitting (divisão de código) e previu dois recursos futuros: concurrent mode e Suspense. Brian demonstrou também como usar a nova ferramenta [React Profiler](/blog/2018/09/10/introducing-the-react-profiler.html) para fazer com que os aplicativos criados no React sejam executados mais rapidamente.

Reviewer: ```suggestion
Na manhã do segundo dia, Andrew Clark e Brian Vaughn apresentaram Concurrent Rendering in React (Renderização Concorrente em React). Andrew abordou a API recentemente anunciada [React.lazy](/blog/2018/10/23/react-v-16-6.html) para code splitting (divisão de código) e previu dois recursos futuros: concurrent mode e Suspense. Brian demonstrou também como usar a nova ferramenta [React Profiler](/blog/2018/09/10/introducing-the-react-profiler.html) para fazer com que os aplicativos criados no React sejam executados mais rapidamente.
```

<br>

<iframe width="560" height="315" src="https://www.youtube.com/embed/UcqRXTriUVI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

> À tarde, Parashuram N falou em detalhes sobre a Nova Arquitetura do React Native, um projeto de longo prazo no qual a equipe React Native tem trabalhado no ano passado e [anunciado em junho](https://facebook.github.io/react-native/blog/2018/06/14/state-of-react-native-2018). Estamos muito entusiasmados com o potencial deste projeto para melhorar o desempenho, simplificar a interoperabilidade com outras bibliotecas e estabelecer uma base sólida para o futuro do React Native.

Reviewer: ```suggestion
À tarde, Parashuram N falou em detalhes sobre a Nova Arquitetura do React Native, um projeto de longa data no qual a equipe React Native esteve trabalhando ao longo do ano passado e [foi anunciado em junho](https://facebook.github.io/react-native/blog/2018/06/14/state-of-react-native-2018). Estamos muito entusiasmados com o potencial deste projeto para melhorar o desempenho, simplificar a interoperabilidade com outras bibliotecas e estabelecer uma base sólida para o futuro do React Native.
```

"long-term" nesse contexto se traduziria melhor por "antigo" ou "de longa data" visto que eles já vêem trabalhando no projeto há um tempo.

Dizer que é "de longo prazo" dá a impressão de que vai ser uma coisa pro futuro...

Author: Make sense

> Agora que a conferência terminou, todas as 28 palestras da conferência estão [disponíveis para transmissão on-line](https://www.youtube.com/playlist?list=PLPxbbTqCLbGE5AihOSExAa4wUM-P42EIJ). Há tonaladas de ótimos conteúdos de ambos os dias. Não podemos esperar até o próximo ano!

Reviewer: ```suggestion
Agora que a conferência terminou, todas as 28 palestras da conferência estão [disponíveis para transmissão on-line](https://www.youtube.com/playlist?list=PLPxbbTqCLbGE5AihOSExAa4wUM-P42EIJ). Há toneladas de ótimos conteúdos de ambos os dias. Mal podemos esperar até o próximo ano!
```

Só pra expressão idiomática do final ficar mais regionalizada...

Author: Isso! eu tinha pensado a mesma coisa... mas acabei deixando assim.. fiquei bem na dúvida!
